src/frontend/magnify/packages/components/src/components/design-system/Formik/ValuesChange/FormikValuesChange.tsx

```diff
@@ -7,10 +7,12 @@ interface FormikValuesChangeProps {
   debounceTime?: number;
   enableSubmit?: boolean;
   children?: React.ReactNode;
+  forceToSubmit?: boolean;
 }
 
 export const FormikValuesChange: FunctionComponent<FormikValuesChangeProps> = ({
   debounceTime = 800,
+  forceToSubmit = false,
   ...props
```

Comment:
You could add `enableSubmit = true` here to avoid the complicated condition below `props.enableSubmit == null || props.enableSubmit` and use `enableSubmit` instead.